app/RecordIssuerType.php

```diff
@@ -6,5 +6,12 @@
 
 class RecordIssuerType extends Model
 {
-    //
+  // record issuer type name stored in database
+  const BILL_TYPE_NAME =  'billing organization';
+  const BANK_STATEMENT_TYPE_NAME = 'bank';
+
+  public function record_issuer()
+  {
+    $this->hasMany(UserRecordIssuer::class);
```

Comment:
This is not the right relationship. It should be `hasManyThrough`, not `hasMany`. See: https://laravel.com/docs/5.4/eloquent-relationships#has-many-through